eZ/Publish/Core/REST/Server/Input/Parser/Criterion/Field.php

```diff
@@ -24,10 +39,50 @@ class Field extends BaseParser
      *
      * @throws \eZ\Publish\Core\REST\Common\Exceptions\Parser
      *
-     * @return \eZ\Publish\API\Repository\Values\Content\Query\Criterion\Field
+     * @return FieldCriterion
```

Comment:
In PHP doc we prefer fully qualified class names.

For completely new code only aiming for v2 and up, we can start to use php type hints instead and skip phpdoc, and thus not having to have fully qualified name but rather the imported alias.

But for this PR from Support perspective it would be good to merge this on 1.13 and up given size is rather small and the positive impact for REST users is relative big.